lib/cli/index.js

```diff
@@ -66,7 +66,7 @@ function cli(options) {
   var ui = new UI({
     inputStream:  options.inputStream,
     outputStream: options.outputStream,
-    ci:           process.env.CI,
+    ci:           process.env.CI || /^(dumb|emacs)$/.test(process.env.TERM),
```

Comment:
It looks like `process.stdout.isTTY` has more to do with the process pipe type than with the terminal's capabilities, because it behaves differently across platforms. On Windows `process.stdout.isTTY` is true inside the Windows console and false inside emacs eshell and compilation-mode like we might expect for it to work here, but on OSX it is true inside all of them: emacs eshell, compilation-mode, as well as a real terminal.